src/pyproject_fmt/formatter/__init__.py

```diff
@@ -13,8 +13,23 @@ def _perform(parsed: TOMLDocument, opts: PyProjectFmtNamespace) -> None:
     fmt_project(parsed, opts)
 
 
-def format_pyproject(opts: PyProjectFmtNamespace) -> str:
-    text = opts.pyproject_toml.read_text()
+def format_pyproject(text: str, opts: PyProjectFmtNamespace) -> str:
+    r"""
+    Format a string with the equivalent content of a ``pyproject.toml`` file.
+
+    Usage:
+
+    >>> from pyproject_fmt import PyProjectFmtNamespace, format_pyproject
+    >>> text = '[project]\nname = "myproj"\nversion = "0.1.1"'
+    >>> opts = PyProjectFmtNamespace(indent=2)
+    >>> format_pyproject(text, opts)
+    '[project]\nname = "myproj"\nversion = "0.1.1"\n'
```

Comment:
This example here should be kept in sync by the tests (I added `--doctest-modules` to `tox.ini`).